### ISSUE TITLE
Backspace should delete preceding letter straightaway

### ISSUE THREAD

**Issue by MichalKoder** [ollelauribostrom/rebus#675]
## Expected Behavior
Backspace should delete preceding letter directly
...

## Actual Behavior
Backspace needs to be pressed twice in order to delete preceding letter
...

## Steps to Reproduce the Problem

1. Open any rebus
2. Type a letter
3. Press backspace

## Specifications

- Browser: Chrome, Edge

**Comment by developer-diganta** [comment]
Hello! I am a beginner to open source and I think I can solve this issue. Can I work on this?

**Comment by MichalKoder** [comment]
Of course, but I'm not sure there's somebody active here to accept your pull request. I've been waiting for my rebus pull request approvement 4 days now. I'm new to open-source too but also learning git. I believe most of the times any feedback is better than no feedback that's why I'm commenting ;)

**Comment by developer-diganta** [comment]
Guess I too can then make my pull request. Hopefully someone looks into these soon!
